.wp-env.json

```diff
@@ -1,4 +1,7 @@
 {
 	"core": "WordPress/WordPress",
-	"plugins": [ "." ]
+	"plugins": [ ".", "./packages/e2e-tests/plugins" ],
```

Comment:
Why are the test plugins here?

Reply:
So they're activated?

Reply:
We would need to have the `env` separation then, otherwise, this would make the development environment very hard to use.

Reply:
They aren't activated because `env` doesn't see into the directory of plugins. But yeah, the `env` separation is the ultimate goal here.

Reply:
lol, `env`, as in `wp-env`, doesn't see into the directory. so it can't call plugin activate on any of the names there.

Reply:
```js
// Activate all plugins.
	for ( const pluginSource of config.pluginSources ) {
		await dockerCompose.run(
			environment === 'development' ? 'cli' : 'tests-cli',
			`wp plugin activate ${ pluginSource.basename }`,
			options
		);
	}
```

That still runs, right? Will it just fail silently?

Reply:
> Will it just fail silently

yes. I can just add them as a mapping, though, to avoid any confusion.

Reply:
done. also fixed a few other things and added a mu-plugins example